src/components/visibility-by-path.js

```diff
@@ -0,0 +1,12 @@
+AFRAME.registerComponent("visibility-by-path", {
+  schema: {
+    path: { type: "string" }
+  },
+  tick() {
+    const userinput = AFRAME.scenes[0].systems.userinput;
+    const shouldBeVisible = !!userinput.get(this.data.path);
+    if (this.el.object3D.visible !== shouldBeVisible) {
+      this.el.setAttribute("visible", shouldBeVisible);
```

Comment:
Did you determine that you had to use the setAttribute version for networking?

Reply:
I have not yet determined that. I plan to figure out that issue and then decide what to do about all the times we do this in the code. A little nervous to just set the visibility on the object3D until I figure that out.